src/components/ActiveContributorSelected/index.tsx

```diff
@@ -1,67 +1,75 @@
 import useSWR from "swr";
-import React from "react";
+import React, { useState } from "react";
 import fetcher from "./fetcher";
-import {ActiveContributor} from "@site/src/components/ActiveContributorSelected/model";
-import {FormControl, MenuItem, Select, SelectChangeEvent} from "@mui/material";
-import lodash from 'lodash';
+import { ActiveContributor } from "@site/src/components/ActiveContributorSelected/model";
+import {
+  FormControl,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from "@mui/material";
+import lodash from "lodash";
 
 export default function ActiveContributorSelected(): JSX.Element {
-    const {
-        data: activeContributors
-    } = useSWR<ActiveContributor[]>('active-contributor-selected', fetcher);
-
-    const [selectMonth, setSelectMonth] = React.useState('');
-
-    let payload = <>Loading...</>
-    if (activeContributors) {
-        const activeMonths = lodash.uniq(activeContributors.map(line => line.activeMonth).sort());
-        const localSelectMonth = selectMonth ? selectMonth : activeMonths[0];
-        const dropDownList = <FormControl sx={{m: 1, minWidth: 120}}>
-            <Select
-                value={localSelectMonth}
-                onChange={(event: SelectChangeEvent) => {
-                    setSelectMonth(event.target.value);
-                }}
-                displayEmpty
-                inputProps={{'aria-label': 'Without label'}}
-            >
-                {activeMonths.map(line => (
-                    <MenuItem value={line}>{line}</MenuItem>
-                ))}
-            </Select>
-        </FormControl>;
-        const selectedContributors = <table>
-            <thead>
+  const { data, error } = useSWR<ActiveContributor[]>(
+    "active-contributor-selected",
+    fetcher
+  );
+  const isLoading = !data && !error;
+  const activeContributors = data || [];
+  const activeMonths = lodash.uniq(
+    activeContributors.map((line) => line.activeMonth).sort()
+  );
+  const [selectMonth, setSelectMonth] = useState(activeMonths[0]);
```

Comment:
If `activeMonths == []`, does this statement throw exception?

Reply:
No, reading any index from an empty array will return `undefined` in js.

Reply:
Thanks for your clarification!

I merge #4 just now. You can review whether there still need some improvements.